util/afero.go

```diff
@@ -89,19 +88,20 @@ func stringify(fs afero.Fs, name string, indent []bool) (string, error) {
 // currently undefined.
 // TODO: this should be made more rsync like and robust!
 func CopyFs(srcFs, dstFs afero.Fs, src, dst string, force bool) error {
-	if src == "" {
-		src = "/"
-	}
-	if dst == "" {
-		dst = "/"
-	}
+	src = path.Join("/", src)
```

Comment:
This is slightly different from the earlier version, but maybe it's better! Will test shortly :)